trade-finance-manager-ui/server/types/express-session.d.ts

```diff
@@ -7,5 +8,8 @@ export type UserSessionData = {
 
 // https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/express-session/index.d.ts#L199-L211
 declare module 'express-session' {
-  interface SessionData extends UserSessionData {}
+  interface SessionData extends UserSessionData {
+    addPaymentErrorKey: AddPaymentErrorKey;
+    checkedCheckboxIds: Record<string, true | undefined>;
+  }
```

Comment:
I'm not sure if these fields should be on the base type as that would apply for all endpoints?

Reply:
Fields here don't need to be optional because the type of the request session is typed as: `session.Session & Partial<session.SessionData>;`